unity3d/Assets/Swrve/SwrveSDK/SwrveSDK.cs

```diff
@@ -432,7 +432,11 @@ public void SessionEnd ()
     public virtual void NamedEvent (string name, Dictionary<string, string> payload = null)
     {
 #if SWRVE_SUPPORTED_PLATFORM
-        NamedEventInternal (name, payload);
+        if (name != null && !name.ToLower().StartsWith("swrve.")) {
+            NamedEventInternal (name, payload);
+        } else {
+            SwrveLog.LogError ("Event cannot begin with \"Swrve.\". The event " + name + " will not be sent");
```

Comment:
I had the lowercase originally, but changed it because I thought you wanted the actual restricted name printed in error message for clarification. The difference being that swrve.* and Swrve.* are not the same.